Libplanet/Blocks/InvalidBlockTransactionsException.cs

```diff
@@ -0,0 +1,13 @@
+using System;
+
+namespace Libplanet.Blocks
+{
+    [Serializable]
+    public class InvalidBlockTransactionsException : InvalidBlockException
+    {
+        public InvalidBlockTransactionsException(string message)
+            : base(message)
```

Comment:
Can't this constructor be omitted?

Reply:
How can we omit it? If we omitted it, we can see a message like below from compiler.

```
InvalidBlockTransactionsException.cs(6, 18): [CS7036] There is no argument given that corresponds to the required formal parameter 'message' of 'InvalidBlockException.InvalidBlockException(string)'
```

Reply:
Nevermind.  I forgot that C# requires exceptions to explicitly declare their own constructor.